javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentStarterImpl.java

```diff
@@ -62,12 +68,44 @@ public void start() {
     ClassLoader savedContextClassLoader = Thread.currentThread().getContextClassLoader();
     try {
       Thread.currentThread().setContextClassLoader(extensionClassLoader);
-      AgentInstaller.installBytebuddyAgent(instrumentation);
+      internalStart();
     } finally {
       Thread.currentThread().setContextClassLoader(savedContextClassLoader);
     }
   }
 
+  private void internalStart() {
+    List<LoggingCustomizer> loggingCustomizers = load(LoggingCustomizer.class);
+    if (loggingCustomizers.size() > 1) {
+      throw new IllegalStateException(
+          "More than one LoggingCustomizerProvider found: "
+              + loggingCustomizers.stream()
+                  .map(Object::getClass)
+                  .map(Class::getName)
+                  .collect(Collectors.joining(", ")));
+    }
+    LoggingCustomizer loggingCustomizer;
+    if (loggingCustomizers.isEmpty()) {
+      loggingCustomizer = new DefaultLoggingCustomizer();
+    } else {
+      loggingCustomizer = loggingCustomizers.get(0);
+    }
+    loggingCustomizer.init();
+
+    Throwable startupError = null;
+    try {
+      ConfigInitializer.initialize();
+      AgentInstaller.installBytebuddyAgent(instrumentation, Config.get());
+    } catch (Throwable t) {
+      startupError = t;
```

Comment:
Should this error be thrown again?

Reply:
I added a comment, I'd like to not rethrow it since that will cause it to be logged again in `OpenTelemetryAgent`, let me know what you think

Reply:
Sounds good, thanks!